### DISCUSSION TITLE
Seems like our school's domain is wrong!

### DISCUSSION THREAD

**Discussion by Juicyburger1250** [community/community#59631] [GitHub Education]
### Hi everyone,

I'm first grade student in Chungbuk science highschool ([https://school.cbe.go.kr/cbs-h](url)).
Our school uses two domains:

- @cbs-h.cbe.go.kr (uses Chungbuk local education office mailing system, but assigned only for our school)
- @cberi.go.kr (this one is for every students in school associated to Chungbuk local education office)

the first one isn't verified, and second one is verified as another highschool near ours.
I've already tried 7times using both domains, but it doesn't seem to work.
It would be appreciated if the email domain is either verified or changed!

**Comment by samus-aran** [comment]
Hi @Juicyburger1250  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
